### DISCUSSION TITLE
GitHub does not recognize BSD-4-Clause LICENSE files

### DISCUSSION THREAD

**Discussion by ultasun** [community/community#30274] [Other Feature Feedback, Questions, & Ideas]
Hello world,

The *BSD 4-Clause License* (also known as the *original BSD License*) does not seem to be recognized after uploading such a `LICENSE` file in a repository.  GitHub only gives me an offer to "View License", or if there are multiple `LICENSE` files in the repository, then it will directly state the `LICENSE` is *Unknown*.

Here is some information about BSD-4-Clause: 
https://spdx.org/licenses/BSD-4-Clause.html
https://en.wikipedia.org/wiki/BSD_licenses#4-clause_license_(original_%22BSD_License%22)

Although it is old, BSD-4-Clause is not an obscure `LICENSE`. 

Since the 4-clause license has the controversial "advertising clause", that seems like a good reason to ensure GitHub is able to identify repositories constrained under this license, in order to help users and developers to be aware of what they are using or modifying. 

Is there any chance that GitHub could add the ability to recognize this `LICENSE` type?

Thank you.

[LICENSE.txt](https://github.com/community/community/files/9388830/LICENSE.txt) (generic template from spdx.org)
[LICENSE.txt](https://github.com/community/community/files/9388831/LICENSE.txt) (a version I currently use for some of my repositories)

**Comment by ccodere** [comment]
Agreed with this comment, some of my old code that i wish to place on github uses BSD 4-clause licenses.
